### ISSUE TITLE
Crash on using a theme that used to work (linux)

### ISSUE THREAD

**Issue by maxlefou** [UltraStar-Deluxe/USDX#370]
### Actual behaviour
If a custom theme (based on the Modern theme with a couple of images and videos changed) has been set, usdx cannot start at all, giving this error in the logs: 
```
An unhandled exception occurred at $000000000042B53A:
EInOutError: Invalid file handle
  $000000000042B53A
```

### Expected behaviour
USDX should start with no problems with the defined theme

### Steps to reproduce
- Go to the game settings
- Set the custom theme
- Note that the game still works perfectly with the theme. No errors, no crashes, nothing, it works as expected.
- Close USDX
- Start USDX again
- The error occurs
- To get rid of the error, edit `config.ini` and copypaste in the `[Themes]` section:
```
Theme=Modern
Skin=Blue
Color=Red
```

### Details
- The custom theme I mention is 99% based on the Modern theme of USDX, but from an old stable version of it. (at least a year old) Were there any critical changes in the themes management that can cause this crash?
- The custom theme is installed in the installation folder set with AUR (`/usr/local/share/ultrastardx`) for the simple reason that the themes folder inside `~/.ultrastardx` is ignored (is it normal?)

- USDX version: the actual master branch of the git repo
- Operating System + version: Manjaro Linux 4.14.39-1

**Comment by basisbit** [comment]
Yes, themes changed quite a bit for the editor and some settings. Easiest would be to diff the txt files to see which parts are missing /invalid in your custom theme

**Comment by basisbit** [comment]
- [ ] TODO: display a helpful error message if theme loading or screen loading failed
